security-jpa-quickstart/README.md

```diff
@@ -47,7 +49,8 @@ _NOTE:_ Stop the database using: `docker-compose down; docker-compose rm`
 
 ### Integration testing
 
-We have provided integration tests based on [TestContainers](https://www.testcontainers.org) to verify the security configuration in a JVM and native  mode.
+We have provided integration tests based on [Dev Services for Postgresql](https://quarkus.io/guides/dev-services#databases) to verify the security configuration in a JVM and native  mode. The test and devmode containers will be launched automatically because all the `Postgresql` configuration properties are only enabled in a production (`prod`) profile.
```

Comment:
```suggestion
We have provided integration tests based on [Dev Services for PostgreSQL](https://quarkus.io/guides/dev-services#databases) to verify the security configuration in JVM and native modes. The test and dev modes containers will be launched automatically because all the PostgreSQL configuration properties are only enabled in production (`prod`) mode.
```